source/install/ee-install.rst

```diff
@@ -4,43 +4,46 @@
+Mattermost Enterprise Edition is free to use in "team mode" without enterprise features enabled. There are two variants of Enterprise Edition, E10 and E20, and you can `compare their features online <https://mattermost.com/pricing-feature-comparison/>`__.
+
+Once you've downloaded and installed Mattermost, start an Enterprise E20 trial via **Main Menu > System Console > Edition and License > Start trial**. You can also create an account in the `Customer Portal <https://customers.mattermost.com/login/>`__, purchase a license key, and upload it to Mattermost via **System Console > Edition and License**.  
 
 Installing Enterprise Edition
 -----------------------------
 
 To install Mattermost Enterprise Edition directly please use one of the following guides:
 
-1. `Production Enterprise Edition on Ubuntu 16.04 <https://docs.mattermost.com/install/install-ubuntu-1604.html>`__
-2. `Production Enterprise Edition on Ubuntu 18.04 <https://docs.mattermost.com/install/install-ubuntu-1804.html>`__
-3. `Production Kubernetes Deployment <https://docs.mattermost.com/install/install-kubernetes.html>`__
-4. `Production Enterprise Edition on RHEL 7 <https://docs.mattermost.com/install/install-rhel-7.html>`__
-5. `Production Enterprise Edition on RHEL 6 <https://docs.mattermost.com/install/install-rhel-6.html>`__
-6. `Production Enterprise Edition on Debian Stretch <https://docs.mattermost.com/install/install-debian.html>`__
-7. `Production Docker Deployment using Docker Compose <https://docs.mattermost.com/install/prod-docker.html>`__
+* `Production Kubernetes Deployment <https://docs.mattermost.com/install/install-kubernetes.html>`__
+* `Production Enterprise Edition on Ubuntu 16.04 <https://docs.mattermost.com/install/install-ubuntu-1604.html>`__
+* `Production Enterprise Edition on Ubuntu 18.04 <https://docs.mattermost.com/install/install-ubuntu-1804.html>`__
+* `Production Enterprise Edition on RHEL 7 <https://docs.mattermost.com/install/install-rhel-7.html>`__
+* `Production Enterprise Edition on RHEL 6 <https://docs.mattermost.com/install/install-rhel-6.html>`__
+* `Production Enterprise Edition on Debian Stretch <https://docs.mattermost.com/install/install-debian.html>`__
+* `Production Docker Deployment using Docker Compose <https://docs.mattermost.com/install/prod-docker.html>`__
 
 Upgrading to Mattermost Enterprise Edition
 -------------------------------------------------
 
 Checking your Mattermost Edition and Version
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you're already running Mattermost, to check your Mattermost edition and version from the server run:
+If you're already running Mattermost, you can check Mattermost edition and version from the command line using:
```

Comment:
should this be "the Mattermost edition and version"

Reply:
Updated.